lib/series_seasonal_adjustment.rb

```diff
@@ -5,12 +5,12 @@ def apply_seasonal_adjustment(factor_application)
     ns_series = Series.get ns_series_name
     raise SeasonalAdjustmentException.new if ns_series.nil?
     set_factors factor_application 
-    new_ns_values = ns_series.get_values_after last_demetra_datestring
+    new_ns_values = ns_series.get_values_after (Date.parse last_demetra_datestring)
```

Comment:
I'll create a new migration to rename `last_demetra_datestring` to `last_demetra_date`